Geometry/HGCalCommonData/plugins/DDHGCalWaferAlgo.cc

```diff
@@ -37,22 +37,20 @@ void DDHGCalWaferAlgo::initialize(const DDNumericArguments & nArgs,
   angles      = vArgs["Angles"];
   detectorType= dbl_to_int(vArgs["DetectorType"]);
 #ifdef EDM_ML_DEBUG
-  edm::LogInfo("HGCalGeom") << childNames.size() << " children: "
-			    << childNames[0] << "; " << childNames[1]
-			    << " positioned " << positionX.size() 
-			    << " times with cell size " << cellSize;
+  std::cout << childNames.size() << " children: " << childNames[0] << "; "
+	    << childNames[1] << " positioned " << positionX.size() 
+	    << " times with cell size " << cellSize << std::endl;
   for (unsigned int k=0; k<positionX.size(); ++k)
-    edm::LogInfo("HGCalGeom") << "[" << k << "] x " << positionX[k] << " y " 
-			      << positionY[k] << " angle " << angles[k]
-			      << " detector " << detectorType[k];
+    std::cout << "[" << k << "] x " << positionX[k] << " y "  << positionY[k]
+	      << " angle " << angles[k] << " detector " << detectorType[k]
+	      << std::endl;
 #endif
   rotns       = sArgs["RotNameSpace"];
   idNameSpace = DDCurrentNamespace::ns();
   parentName  = parent().name(); 
 #ifdef EDM_ML_DEBUG
-  edm::LogInfo("HGCalGeom") << "DDHGCalWaferAlgo debug: Parent " 
-			    << parentName << " NameSpace " << idNameSpace
-			    << " for Rotation " << rotns;
+  std::cout << "DDHGCalWaferAlgo debug: Parent " << parentName << " NameSpace "
```

Comment:
@bsunanda - the code would benefit from replacing std::cout with
```
LogDebug("DDHGCalWaferAlgo") << ...
```
it will allow also to remove the lines:
```
#ifdef EDM_ML_DEBUG
...
#endif
```